src/reflection.ts

```diff
@@ -463,6 +465,13 @@ export class Parser {
     return (t: Table) => {
       const offset = t.offset + t.bb.__offset(t.offset, field.offset());
       if (offset === t.offset) {
+        // If the field is marked as 'optional' in the schema, return `null` when it is absent.
```

Comment:
nit: Can you add a parenthetical, since I at least am not actually used to using `optional` for these things:

`// If the field is marked as 'optional' in the schema (i.e., the 'default' is `null`), return `null`....`